tests/foreman/api/test_syncplan.py

```diff
@@ -74,1024 +78,994 @@ def valid_sync_interval():
     return ['hourly', 'daily', 'weekly', 'custom cron']
 
 
-class SyncPlanTestCase(APITestCase):
-    """Miscellaneous tests for sync plans."""
+@tier1
+def test_positive_get_routes():
+    """Issue an HTTP GET response to both available routes.
 
-    @tier1
-    def test_positive_get_routes(self):
-        """Issue an HTTP GET response to both available routes.
+    :id: 9e40ea7f-71ea-4ced-94ba-cde03620c654
 
-        :id: 9e40ea7f-71ea-4ced-94ba-cde03620c654
+    :expectedresults: The same response is returned.
 
-        :expectedresults: The same response is returned.
+    :BZ: 1132817.
 
-        Targets BZ 1132817.
+    :CaseImportance: Critical
+    """
+    org = entities.Organization().create()
+    entities.SyncPlan(organization=org).create()
+    response1 = client.get(
+        '{0}/katello/api/v2/sync_plans'.format(settings.server.get_url()),
+        auth=settings.server.get_credentials(),
+        data={'organization_id': org.id},
+        verify=False,
+    )
+    response2 = client.get(
+        '{0}/katello/api/v2/organizations/{1}/sync_plans'.format(
+            settings.server.get_url(), org.id
+        ),
+        auth=settings.server.get_credentials(),
+        verify=False,
+    )
+    for response in (response1, response2):
+        response.raise_for_status()
+    assert response1.json()['results'] == response2.json()['results']
 
-        :CaseImportance: Critical
-        """
-        org = entities.Organization().create()
-        entities.SyncPlan(organization=org).create()
-        response1 = client.get(
-            '{0}/katello/api/v2/sync_plans'.format(settings.server.get_url()),
-            auth=settings.server.get_credentials(),
-            data={'organization_id': org.id},
-            verify=False,
-        )
-        response2 = client.get(
-            '{0}/katello/api/v2/organizations/{1}/sync_plans'.format(
-                settings.server.get_url(), org.id
-            ),
-            auth=settings.server.get_credentials(),
-            verify=False,
-        )
-        for response in (response1, response2):
-            response.raise_for_status()
-        self.assertEqual(response1.json()['results'], response2.json()['results'])
 
+@pytest.mark.parametrize("enabled", [False, True])
+@tier1
+def test_positive_create_enabled_disabled(module_org, enabled):
+    """Create sync plan with different 'enabled' field values.
 
-class SyncPlanCreateTestCase(APITestCase):
-    """Tests specific to creating new sync plans."""
+    :id: df5837e7-3d0f-464a-bd67-86b423c16eb4
 
-    @classmethod
-    def setUpClass(cls):
-        """Create an organization which can be re-used in tests."""
-        super(SyncPlanCreateTestCase, cls).setUpClass()
-        cls.org = entities.Organization().create()
+    :expectedresults: A sync plan is created, 'enabled' field has correct
```

Comment:
Add :parametrized: yes before expected results in all the parameterized test cases.

Reply:
OK, done. Thank you for review.